packages/cli/package.json

```diff
@@ -21,10 +21,8 @@
     "@types/fs-extra": "^11.0.1",
     "@types/gunzip-maybe": "^1.4.0",
     "@types/prettier": "^2.7.2",
-    "@types/recursive-readdir": "^2.2.1",
-    "@types/stack-trace": "^0.0.30",
```

Comment:
So this is what you were saying yesterday, all of these deps can be removed from the CLI, which will make `npm create @shopify/hydrogen` faster? And instead they are deps to the new package, which is a peer dep.

Reply:
Yeah. The main dependency here is `miniflare` (123MB): https://pkg-size.dev/miniflare@3
Now this is only installed when doing `npm i` in the template, not when the CLI is installed globally with `npm create @shopify/hydrogen`.